loggingpy/log.py

```diff
@@ -180,6 +181,9 @@ def log(self, log_entry: LogEntry):
         self.logger.log(log_entry.log_level.value, "", extra={'log_entry': log_entry})  # exc_info=True, stack_info=True, add this to drop out some dto info
 
     def write_entry(self, log_level: Enum, payload_type: str, data=None, exception: Exception = None):
+
+        if self.prefix_payload_type and self.context is not None and self.context is not "":
+            payload_type = self.context + '.' + payload_type
         log_entry = LogEntry(context="", log_level=log_level, payload_type=payload_type, data=data, exception=exception)
```

Comment:
out of interest: what is the empty `context` of `LogEntry`?

Reply:
As far as I can see, you do the same in your logger.

    LogEntry entry = new LogEntry
            {
                LogLevel = level,
                PayloadType = payloadType,
                Data = data,
                Exception = exception
            };

I am not sure why the context of the logentry is never set and thus always overridden by the logger's context.